### DISCUSSION TITLE
Feature Request: PDAL

### DISCUSSION THREAD

**Discussion by BruceHarold** [duckdb/duckdb#11581] [General]
Hi, I may have missed this but spatial support for [PDAL](https://pdal.io/en/2.7.1/index.html) would be another string to Duck's bow.

**Comment by Tishj** [comment]
That sounds like it would be suited for Spatial, am I wrong in that assumption?

**Reply by BruceHarold** [reply]
That's the idea, add it to the spatial extension.  The main use case is cloud optimized point cloud (COPC).
